### ISSUE TITLE
GIF Treatment

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#677]
## Bug Description

In the media library panel, GIFs are being filtered under videos. This makes sense to me but there was criteria to treat gifs as images for now, do we need to change this filter behaviour? 

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by spacedmonkey** [comment]
This is now happening for me

**Comment by pbakaus** [comment]
This is working....~ish. A couple things:

1. I just tried going to https://giphy.com/gifs/moodman-coronavirus-tp-toilet-paper-H3NXbLWLwv4b52X4ih, right clicked on the image, clicked "copy image"
2. Then went back to the editor, selected the workspace, and hit "CMD + V".
3. After waiting a while without any indication that it's uploading or doing anything (we really need to fix this), a static image appeared.
4. I confirmed in the media library that it was uploaded as png (?!)
5. Saving the gif directly to my Desktop, then dragging it into the media library worked
